debian/patches/series

```diff
@@ -1,5 +1,4 @@
 # Backported from upstream.
-cldr-34-update.patch
```

Comment:
You cannot remove the CLDR-34 patch, we backported this specifically to support other locales than what was shipped in 5.9

Reply:
Ok I'll try to update the patch to fix the build issues